docs/openapi/schemas.yaml

```diff
@@ -261,22 +261,22 @@ Site:
         'cwv':
           disabled: false
     config:
-        slack:
-          channel: 'C1234567890'
-          workspace: 'example-workspace'
-        alerts:
-          - type: '404'
-            byOrg: true
-            mentions:
-              - slack:
-                  - '@user1'
-                  - '@user2'
-          - type: 'cwv'
-            byOrg: false
-            mentions:
-                - slack:
-                    - '@user5'
-                    - '@user6'
+      slack:
+        channel: 'C1234567890'
+        workspace: 'example-workspace'
+      alerts:
+        - type: '404'
+          byOrg: true
```

Comment:
aside from the alignment fix: why is the `byOrg` set on a config for the site? Shouldn't it be on the one for the organization?

Reply:
@alinarublea i can't remember - do you know?